webapp/components/test-file-results.html

```diff
@@ -205,21 +205,22 @@
       }
 
       async loadResultFile(testRun) {
-        const url = this.resultsURL(testRun);
+        const url = this.resultsURL(testRun, this.path);
         const response = await window.fetch(url);
         if (!response.ok) {
           return null;
         }
         return response.json();
       }
 
-      resultsURL(testRun) {
+      resultsURL(testRun, path) {
+        path = this.encodeTestPath(path);
```

Comment:
This should be done after the if statement for the `diff` case. Otherwise, the path will be double-encoded.

Reply:
FYI I directly fixed it myself.